k8s-logging-cfg.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-cfg
+  namespace: default
+data:
+  filebeat.yml: |
+    filebeat.registry_file: /var/log/containers/filebeat_registry
+
+    filebeat.prospectors:
+    - input_type: log
+      paths:
+        - "/var/lib/docker/containers/*/*.log"
+      symlinks: true
+      json.keys_under_root: true
+      json.add_error_key: true
+      json.message_key: log
+      multiline.pattern: '^\s'
+      multiline.match: after
+      document_type: kube-logs   
+
+    output.logstash:
+      hosts: ${LOGSTASH_HOSTS:?No logstash host configured. Use env var LOGSTASH_HOSTS to set hosts.}
+      timeout: 15
+    # Available log levels are: critical, error, warning, info, debug
+    logging.level: info
```

Comment:
Can you take this config from `filebeat.yml` in the repo to keep it as default config?

Reply:
done